lib/move/__tests__/move.test.js

```diff
@@ -5,7 +5,6 @@ const os = require('os')
 const fse = require(process.cwd())
 const path = require('path')
 const assert = require('assert')
-const rimraf = require('rimraf')
```

Comment:
Is `rimraf` used elsewhere, or can we remove it as a devDependency?

Reply:
It is used in a few test files, but we definitely can remove it from our devDependencies since `rimraf` essentially exists within `remove`. In addition, there are 2 more devDependencies that I believe are not needed and can be removed: `read-dir-files` and `secure-random`. We'll take care of them later!